### ISSUE TITLE
Upgrade from 1.9.2 to 1.9.5.1

### ISSUE THREAD

**Issue by cervedgroup** [osTicket/osTicket-1.8#1696]
Just upgraded, cannot see anymore the "priority" in tickets list
![noprior](https://cloud.githubusercontent.com/assets/6535381/5874245/9098752e-a2ff-11e4-818d-1f76a8e18cb7.JPG)

**Comment by ntozier** [comment]
Works fine in my installations.  Please check your error logs.

**Comment by cervedgroup** [comment]
found some 1062 errors : 

DB Error #1062
[INSERT INTO ost_ticket_status SET id = 1, name = 'Open', state = 'open', mode = 3, sort = 1, properties = '{\"description\":\"Open tickets.\"}', created = NOW()] Duplicate entry '1' for key 'PRIMARY'

 ---- Backtrace ----
 #0 C:\Osticket\upload\include\mysqli.php(177): osTicket->logDBError('DB Error #1062', '[INSERT INTO os...')
 #1 C:\Osticket\upload\include\class.orm.php(221): db_query('INSERT INTO ost...')
 #2 C:\Osticket\upload\include\class.list.php(1143): VerySimpleModel->save(true)
 #3 C:\Osticket\upload\include\upgrader\streams\core\8f99b8bf-03ff59bf.task.php(31): TicketStatus::__create(Array)
 #4 C:\Osticket\upload\include\class.upgrader.php(372): SequenceLoader->run(30)
 #5 C:\Osticket\upload\include\class.upgrader.php(122): StreamUpgrader->doTask()
 #6 C:\Osticket\upload\include\ajax.upgrader.php(35): Upgrader->doTask()
 #7 [internal function]: UpgraderAjaxAPI->upgrade()
 #8 C:\Osticket\upload\include\class.dispatcher.php(145): call_user_func_array(Array, Array)
 #9 C:\Osticket\upload\include\class.dispatcher.php(38): UrlMatcher->dispatch('/upgrader', NULL)
 #10 C:\Osticket\upload\scp\ajax.php(186): Dispatcher->resolve('/upgrader')
 #11 {main}

DB Error #1062
[INSERT INTO ost_sequence SET id = 1, name = 'General Tickets', next = 1, padding = '0', increment = 1, flags = 1] Duplicate entry '1' for key 'PRIMARY'

 ---- Backtrace ----
 #0 C:\Osticket\upload\include\mysqli.php(177): osTicket->logDBError('DB Error #1062', '[INSERT INTO os...')
 #1 C:\Osticket\upload\include\class.orm.php(221): db_query('INSERT INTO ost...')
 #2 C:\Osticket\upload\include\upgrader\streams\core\8f99b8bf-03ff59bf.task.php(16): VerySimpleModel->save()
 #3 C:\Osticket\upload\include\class.upgrader.php(372): SequenceLoader->run(30)
 #4 C:\Osticket\upload\include\class.upgrader.php(122): StreamUpgrader->doTask()
 #5 C:\Osticket\upload\scp\upgrade.php(39): Upgrader->doTask()
 #6 {main}

[INSERT INTO ost_sequence SET id = 1, name = 'General Tickets', next = 1, padding = '0', increment = 1, flags = 1] Duplicate entry '1' for key 'PRIMARY'

 ---- Backtrace ----
 #0 C:\Osticket\upload\include\mysqli.php(177): osTicket->logDBError('DB Error #1062', '[INSERT INTO os...')
 #1 C:\Osticket\upload\include\class.orm.php(221): db_query('INSERT INTO ost...')
 #2 C:\Osticket\upload\include\upgrader\streams\core\8f99b8bf-03ff59bf.task.php(16): VerySimpleModel->save()
 #3 C:\Osticket\upload\include\class.upgrader.php(372): SequenceLoader->run(30)
 #4 C:\Osticket\upload\include\class.upgrader.php(122): StreamUpgrader->doTask()
 #5 C:\Osticket\upload\include\ajax.upgrader.php(35): Upgrader->doTask()
 #6 [internal function]: UpgraderAjaxAPI->upgrade()
 #7 C:\Osticket\upload\include\class.dispatcher.php(145): call_user_func_array(Array, Array)
 #8 C:\Osticket\upload\include\class.dispatcher.php(38): UrlMatcher->dispatch('/upgrader', NULL)
 #9 C:\Osticket\upload\scp\ajax.php(186): Dispatcher->resolve('/upgrader')
 #10 {main}

**Comment by greezybacon** [comment]
@cervedgroup both of those tables were new in v1.9.4. Did you have to run the upgrade twice?

**Comment by greezybacon** [comment]
@cervedgroup try dropping the %ticket__cdata table and refresh the ticket queue to see if the priorities magically reappear
